pilot/pkg/bootstrap/server.go

```diff
@@ -410,6 +410,11 @@ func (s *Server) initMesh(args *PilotArgs) error {
 		}
 	}
 
+	if err = model.ValidateMeshConfig(mesh); err != nil {
```

Comment:
This is too dangerous - invalid location-based config should not crash down the entire cluster ( which will happen since pilot will crash loop ).

Please report an error, increment a stat - but make sure pilot keeps running.

Reply:
Isn't this a normal behavior for a system to  stop bootstrap to prevent undefined error once a config is invalid?

Reply:
This is in the bootstrap procedure ,  for a reload procedure  i think we can omit it and keep running use the previous config.